docs/lib/datastore/fragments/android/sync/40-clear.md

```diff
@@ -31,6 +33,23 @@ Amplify.Hub.subscribe(HubChannel.AUTH,
 )
 ```
 
+</amplify-block>
+
+<amplify-block name="RxJava">
+
+```java
+// Listen for sign out events.
+final String signedOutEventName = AuthChannelEventName.SIGNED_OUT.toString();
+
+RxAmplify.Hub.on(HubChannel.AUTH)
+    .filter(event -> signedOutEventName.equals(event.getName()))
+    .flatMapObservable(RxAmplify.DataStore::clear)
```

Comment:
Uh oh, looks we didn't add `clear()` into the Rx bindings. I'm on it. PR to follow...